### ISSUE TITLE
Deploy poolsv2 on testnet

### ISSUE THREAD

**Issue by Paul4912** [tracer-protocol/perpetual-pools-contracts#247]

**Comment by Paul4912** [comment]
Maybe will need to create new issue for problems that arise from this deployment.

**Comment by Paul4912** [comment]
Includes solving issues that pop up and need a pr for as a result of this

**Comment by nisdo** [comment]
@Paul4912 can you please give a status on this or place link to the relevant Discord chat if it's already being discussed elsewhere?

**Comment by mynameuhh** [comment]
@nisdo This was blocked by an issue found and resolved as part of https://github.com/tracer-protocol/perpetual-pools-contracts/pull/273#pullrequestreview-827227860.
I believe @Paul4912 is going to re-deploy once that is fixed which I think is a good idea.

**Comment by nisdo** [comment]
ok, when there are PRs @Paul4912 can you please link them (right hand side menu) in future?
